examples/ex1.cpp

```diff
@@ -24,6 +24,7 @@
 //               ex1 -m ../data/fichera-amr.mesh
 //               ex1 -m ../data/mobius-strip.mesh
 //               ex1 -m ../data/mobius-strip.mesh -o -1 -sc
+//               ex1 -m ../data/inline-quad.mesh -o 3 -ser
 //
```

Comment:
I am not sure we want these changes in Example 1 and 2 -- we are trying to keep them as simple as possible.